src/main/java/cz/lundegaard/form/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package cz.lundegaard.form.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
```

Comment:
V pripade, ze anotace ma field value a je to jediny field, ktery nastavujete, lze pouzit konvenci @ResponseStatus(HttpStatus.NOT_FOUND) - kratsi, trosku citelnejsi, nic zasadniho :)